scripts/building/dev_setup.py

```diff
@@ -40,3 +40,4 @@ def setup_promptflow(extra_deps: list, command_args: dict) -> None:
 
     command_args = {"shell": system() == "Windows", "stream_stdout": args.verbose}
     setup_promptflow(extra_deps=args.promptflow_extra_deps, command_args=command_args)
+    run_command(commands=["pre-commit", "install"], **command_args)
```

Comment:
consider pre-commit update component?

Reply:
offline synced, will try this with another PR